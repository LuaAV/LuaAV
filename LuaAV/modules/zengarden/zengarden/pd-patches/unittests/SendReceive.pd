#N canvas 453 179 442 142 10;
#X obj 16 5 loadbang;
#X obj 16 26 t b b b;
#X obj 62 56 print start;
#X obj 39 84 s distant;
#X obj 16 117 print end;
#X obj 174 12 r distant;
#X obj 326 11 r distant;
#X obj 326 32 print distant2;
#X obj 174 33 t b b;
#X obj 174 76 print distant1-2;
#X obj 206 55 print distant1-1;
#X connect 0 0 1 0;
#X connect 1 0 4 0;
#X connect 1 1 3 0;
#X connect 1 2 2 0;
#X connect 5 0 8 0;
#X connect 6 0 7 0;
#X connect 8 0 9 0;
#X connect 8 1 10 0;
