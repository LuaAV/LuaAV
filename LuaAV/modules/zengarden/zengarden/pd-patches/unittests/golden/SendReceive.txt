start: bang
distant2: bang
distant1-1: bang
distant1-2: bang
end: bang
